### ISSUE TITLE
Launching a program with MOD+space only works after restarting spectrwm

### ISSUE THREAD

**Issue by ghillb** [conformal/spectrwm#429]
**System:** Ubuntu 20.10
**Version:** spectrwm 3.4.1 Build: 3.4.1

After booting into Ubuntu and logging in, it is not possible to launch a program that is bound to `MOD+space`  (I'm using `Mod4`).
Only after restarting spectrwm (in my case MOD+x), this shortcut works and I can launch the program that is bound to `MOD+space` via the binding.

**Steps to reproduce:**
1. Bind a program to MOD+space , e.g. firefox:
```
program[firefox]          = firefox
bind[firefox]             = MOD+space
```

2. Log into a new session
3. Try to launch firefox with `MOD+space`

My spectrwm.conf:
https://raw.githubusercontent.com/ghillb/dotfiles/master/spectrwm.conf

Is there a way to restart spectrwm via a scripted command to get around this issue? Everything else works perfectly.

**Comment by ghillb** [comment]
I do  load a custom keymap on startup via `.profile`. I removed it for testing, but the issue persists with the default keymap.

After reading [#310 (comment)](https://github.com/conformal/spectrwm/issues/310#issuecomment-621158723) I think its a different problem. The keymap loads normally and works everywhere (all keys I remapped work also for spectrwm mappings).
I did not touch space or the super key in the custom key map.

**Comment by LordReg** [comment]
Perhaps some other app (maybe in your startup.sh script) is doing a key grab. do you run a notification daemon? If not, it's likely the keymap is changing after spectrwm starts. To check that, you can add `xmodmap -pke > ~/xmodmap.txt` in your .xinitrc/.xprofile/etc so it is run immediately before `spectrwm`.  The output would be the keymap spectrwm sees when it first starts.  If you run `xmodmap -pke` again in a terminal after spectrwm starts, you can see if it changed.

**Comment by ghillb** [comment]
I tried your approach and realized that the before- and after-output were indeed different. I tried it without the [startup script](https://raw.githubusercontent.com/ghillb/dotfiles/master/assets/startup.sh) and got the same result.

Regardless of what the culprit is, your first advice was correct all along and the issue went away! 
I put `setxkbmap some_map; exec spectrwm` into `.xprofile` and it worked. Yesterday I didn't realize that `.xinitrc` is ignored on my installation.

Thank you!
